src/main/java/org/repodriller/filter/diff/OnlyDiffsWithFileTypes.java

```diff
@@ -0,0 +1,23 @@
+package org.repodriller.filter.diff;
+
+import java.util.List;
+
+/**
+ * Only process diffs on files with certain file extensions.
+ * 
+ * @author Ayaan Kazerouni
+ */
+public class OnlyDiffsWithFileTypes implements DiffFilter {
+
+	private List<String> fileExtensions;
+	
+	public OnlyDiffsWithFileTypes(List<String> fileExtensions) {
```

Comment:
Here you use `List<String>`, earlier you used `DiffFilter...`. Why?

Reply:
Assuming you're only asking about syntax consistency -- `DiffFilter...` collects the arguments into an Array, which we put into a list using `Arrays.asList`. I assume this is for the sake of convenience, so the user can just list filters they want to use as params while setting up the `Study` and not bother with brackets and such.

Within the filter itself, I just followed the syntax in `CommitFilter` classes. Specifically `OnlyModificationsWithFileTypes`.

Reply:
OK.